src/routes/MapExplorer/MapExplorer.connector.js

```diff
@@ -93,18 +98,31 @@ export function mapDispatchToProps (dispatch, props) {
     fetchMembers: (params) => () => dispatch(fetchMembers({ ...params })),
     fetchPosts: (params) => () => dispatch(fetchPosts({ ...params })),
     fetchPublicCommunities: (params) => () => dispatch(fetchPublicCommunities({ ...params })),
+    fetchSavedSearches: (userId) => () => dispatch(fetchSavedSearches(userId)),
+    deleteSearch: (searchId) => dispatch(deleteSearch(searchId)),
+    saveSearch: (params) => dispatch(saveSearch(params)),
     showDetails: (postId) => dispatch(push(postUrl(postId, { ...routeParams, view: 'map' }, querystringParams))),
     showCommunityDetails: (communityId) => dispatch(push(communityMapDetailUrl(communityId, { ...routeParams, view: 'map' }, querystringParams))),
     gotoMember: (memberId) => dispatch(push(personUrl(memberId, routeParams.slug, routeParams.networkSlug))),
     toggleDrawer: (hidden) => dispatch(push(addQuerystringToPath(baseUrl({ ...routeParams, view: 'map' }), { ...querystringParams, hideDrawer: hidden }))),
     storeFetchParams: param => opts => dispatch(storeFetchParams({ ...param, ...opts })),
-    storeClientFilterParams: params => dispatch(storeClientFilterParams(params))
+    storeClientFilterParams: params => dispatch(storeClientFilterParams(params)),
+    viewSavedSearch: (params, path, selectedSearch) => {
+      const { boundingBox, featureTypes, search, topics } = params
+      dispatch(fetchMembers({ ...params }))
+      dispatch(fetchPosts({ ...params }))
+      dispatch(fetchPublicCommunities({ ...params }))
```

Comment:
this smells a bit off to me, seems like viewing a search should just change the params and then the map should call all these on its own since/if the params have changed